ts/components/screens/PinCreation/usePinValidationBottomSheet.tsx

```diff
@@ -0,0 +1,28 @@
+import { VSpacer, Body } from "@pagopa/io-app-design-system";
+import React, { memo } from "react";
+import { View } from "react-native";
+import I18n from "../../../i18n";
+import { BulletList } from "../../BulletList";
+import { useIOBottomSheetModal } from "../../../utils/hooks/bottomSheet";
+
+const BottomSheetContent = memo(() => (
+  <View>
+    <Body>{I18n.t("onboarding.pin.policy.description")}</Body>
+    <VSpacer size={16} />
+    <BulletList
+      title={I18n.t("onboarding.pin.policy.bulletListTitle")}
+      list={[
+        { value: I18n.t("onboarding.pin.policy.firstItem"), id: "first_item" },
+        { value: I18n.t("onboarding.pin.policy.secondItem"), id: "second_item" }
+      ]}
+      spacing={16}
+    />
+  </View>
```

Comment:
I guess here we are missing some sort of bottom margin accounting the bottom safe area view (for example for escaping the home indicator on some iOS devices). What do you think?

Reply:
Are you suggesting to add some margin bottom to avoid this?

Reply:
Yup! We could use `useSafeAreaInsets` and take the bottom margin or padding value or keep it simple and add a `VSpace` of `24`. What do you think?

Reply:
Thank you for the suggestion!